build-tools/docker/check-docker-compose-services.sh

```diff
@@ -0,0 +1,21 @@
+export SAPASSWORD=$(openssl rand -base64 12)
+
+docker-compose up -d
+echo "Waiting for docker compose services..."
+
+is_healthy() {
+    service="$1"
+    container_id="$(docker-compose ps -q "$service")"
+    health_status="$(docker inspect -f "{{.State.Health.Status}}" "$container_id")"
+
+    if [ "$health_status" = "healthy" ]; then
+        echo "$1 is healthy"
+        return 0
+    else
+        echo "Waiting for $1"
+        return 1
+    fi
+}
+
+while ! is_healthy trial-config-service; do sleep 5; done
+while ! is_healthy practitioner-service; do sleep 5; done
```

Comment:
Those would run infinitely, right?
I think we should add some protection here, like max 90 seconds or so...

Reply:
Doh. Meant to that in the action job timeout. Will fix.

Reply:
Marcus is this a property on a job ?

timeout-minutes: 1

Reply:
Yes, that's what is meant to do. 1 minute should be plenty I think?